.pipelines/cni/load-test-templates/pod-deployment-template.yaml

```diff
@@ -26,7 +27,10 @@ steps:
   - template: ../../templates/log-template.yaml
     parameters:
       clusterName: ${{ parameters.clusterName }}
-      logType: scaleTest
       os: ${{ parameters.os }}
       cni: ${{ parameters.cni }}
       jobName: ${{ parameters.jobName }}
+      ${{ if eq(parameters.upgradeScenario, 'true') }}:
+        logType: upgradeScaleTest

```

Comment:
Maybe we pass the `jobname` as `upgrade_scale_test` (Similarly at other places), will that avoid the parameter of `logType` ?